### ISSUE TITLE
Ingest Content for Content Provider - RENNES1

### ISSUE THREAD

**Issue by lobajuluwa** [gbhl/bhl-europe#330]
Task description: Align (RENNES1) upload data/structure with ingest tool needs

subtask: Ingest (RENNES1) data
## Actions to take:

review any non monograph content to come, otherwise upload good 
## Summary:
- folder names - ok
- folder structure - monograph
- file names: 
  - InternalIdentifier - present ok
  - FileSequenceNumber - present ok
  - PageType -present (cover,blank,halftitle,title,index,preface,page,plate)
  - PrintedPageNumber - present
- metadata available - yes
- metadata in accepted format - marc21
- Bibliographic level - monograph

**Comment by melitabirthaelmer** [comment]
Re-uploaded content. Check these folders please:

F352382102_11716
F352382102_46852

Changes:
- translation into English of the page type
- use of the terms listed (sent additional pagetypes, ok from Wolfgang except  for caption; new pagetypes are here:https://bhl.wikispaces.com/Content+_Management+Harmonisation)
- underscore deleted from the library identifier number

**Comment by chris-sleep** [comment]
created fr-rennes1 PI account on int_pi_pi

Generated .aip for content F352382102_46852

@wkollernhm would you mind taking a look please at the .aip and confirm if it all looks good? Provided so we can move ahead with upload for Rennes

**Comment by chris-sleep** [comment]
@ZhengLIAtos. @wkollernhm 

I've regenerated the .aip on bhl-test for the above content (fr-rennes1/F352382102_46852) - can you please run a test ingest (as soon as ingest deploy on integration is good) and confirm if this content is good to go, then we can give the go ahead for upload to our provider.

**Comment by wkollernhm** [comment]
Some of the files do use a wrong naming scheme.

0096: there is a whitespace in the name, the type of the page should be after the sequence
0097: type of page should follow the sequence, not the page-number

examples:
"46852_0096_page  0084_blank.tif" => "46852_0096_page_0084.tif" OR "46852_0096_blank_0084.tif"
"46852_0097_page_0085_index.tif" => "46852_0097_page_0085.tif" OR "46852_0097_index_0085.tif"

of course this leads to invalid API output.

**Comment by melitabirthaelmer** [comment]
CP informed to change wrong naming for F352382102_46852

Please check additionally following folders:
F352382104_18747_1
F352382104_18747_2

Here CP provided pdf files with OCR. Tiff files in same resolution can be provided in addition, if necessary.

**Comment by wkollernhm** [comment]
Those two folders are fine, thought they do not contain any TIFF files anymore. But one PDF including one MARC file is perfectly fine and can be used.

**Comment by chris-sleep** [comment]
running a PI test on F352382104_18747_1

OLEF ok; 

@wkollernhm initial generation of TIF generation reported generating ppm for document (500+ pages) - no console update and timed out, but looks like .ppm created in the .aip folder ok; rerunning the tif step picked those .ppm up and convert ran ok.

I ran a comparison in parallel on 18747_2 (same browser session) - and the first vanished. testing with the text generation, it looks like if you invoke the same step on two content items at the same time, both write out to the same popup, the second overloads the first - bug or design restriction?

**Comment by chris-sleep** [comment]
@ZhengLIAtos I've generated an .aip for  fr-rennes1/F352382104_18747_1 - can you please validate this for ingest (and if it's good push it into ingest so we can review how it comes through to the bhl-int portal?)  Thanks.

**Comment by wkollernhm** [comment]
@chris-sleep This is by design, but we can consider this as not useful and create a own popup for each action. This however might lead to tons of popups...

**Comment by chris-sleep** [comment]
CP has corrected F352382102_46852 files, now regenerating the .aip

**Comment by chris-sleep** [comment]
@ZhengLIAtos can you please also run the ingest on fr-rennes1/F352382102_46852 (as well as fr-rennes1/F352382104_18747_1 above)  to confirm how this looks?

**Comment by melitabirthaelmer** [comment]
Following folders have been uploaded by CP and can be ingested:
F352382102_41266
F352382102_41267
F352382102_46890vol2
F352382102_76103
F352382104_30389
F352382104_175509T
F352382104_E732vol17
F352382104_E732vol25

**Comment by melitabirthaelmer** [comment]
Following folders have been uploaded by CP and can be ingested:
F352382102_41266
F352382102_41267
F352382102_46890vol2
F352382102_76103
F352382104_30389
F352382104_175509T
F352382104_E732vol17
F352382104_E732vol25

**Comment by HenningScholz** [comment]
PI is running for all folders except F352382104_18747_1/ and F352382104_18747_2/ (because they have pdf) and F352382104_175512/ (because it is a serial that needs to be checked). As these are only comparable low number of pages and special problems to solve, we consider them as low priority and focus on other stuff first.
